scripts/ci/check-connect-data.ts

```diff
@@ -81,6 +81,49 @@ const updateConfigFromJSON = async () => {
         if (changes.stdout !== '') {
             console.log('There were changes in keys.');
 
+            // Before creating the new PR with new keys we check if there was already previous one.
+            // We can delete the previous one since latest one will contain all new changes.
+            // If you need to update search query you can test in GH: https://github.com/search and
+            // use the documentation https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests
+            const prList = await exec('gh', [
+                'search',
+                'prs',
+                '--repo=trezor/trezor-suite',
+                '--head=chore/update-device-authenticity-config',
+                '--state=open',
+            ]);
+
+            console.log('prList', prList);
+
+            if (prList.stdout !== '') {
+                const prNumbers = prList.stdout.match(/(?<=\t)(\d+)(?=\t)/g);
+
+                console.log(`Found open pull requests ${prNumbers}. Closing...`);
+                for (const prNumber of prNumbers) {
```

Comment:
I guess after this is merged we can always (probably) expect only one record in prNumbers, right? but it is always better to expect the worst case scenario

Reply:
Yes, you are right. If everything works well we should never see more than one of those PRs open.